src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -65,13 +68,15 @@ export const RateGasRow: FC<RateGasRowProps> = memo(
     swapperName,
     swapSource,
     networkFeeFiatUserCurrency,
+    deltaPercentage,
     onClick,
     noExpand,
-    isOpen: defaultIsOpen,
+    invertRate = false,
   }) => {
     const translate = useTranslate()
-    const { isOpen, onToggle } = useDisclosure({ defaultIsOpen })
-
+    const { isOpen, onToggle } = useDisclosure()
+    const [shouldInvertRate, setShouldInvertRate] = useState(invertRate)
+    const hasClickedRate = useRef(false)
```

Comment:
q: could this just be a state field?

Reply:
Definitely, I thought it would be safer to use a ref, but by testing it seems like working fine without

Changed in [2fc1511](https://github.com/shapeshift/web/pull/8970/commits/2fc15110e25204d7f59b309c7b574a6c6b83118d)